src/components/Popup.js

```diff
@@ -3,19 +3,21 @@ export default class Popup {
         this._popup = document.querySelector(popupSelector);
     }
 
-    open() {
+    open(){
         this._popup.classList.add("modal_opened");
-        document.addEventListener("keydown", this._handleEscClose);
-    };
+        document.addEventListener("keydown",
+            this._handleEscClose);
+           }

```

Comment:
You need to format the markup (and code) to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`


```js

  open() {
    this._popup.classList.add("modal_opened");
    document.addEventListener("keydown", this._handleEscClose);
  }


```